include/seqan/rna_io/rna_record.h

```diff
@@ -0,0 +1,289 @@
+     */
+    CharString comment;
+
+    /*!
+     * @var StringSet<String<float>> RnaRecord::reactivity
+     * @brief The area peak/likelihood estimate that represents the <b>reactivity</b> at each position.
+     *
+     * This member variable is used only if biological validated data is found (T.. fields in EBPSEQ are set).
+     */
+    StringSet<String<float> > reactivity;
+
+    /*!
+     * @var StringSet<String<float>> RnaRecord::reactError
+     * @brief Error of the @link RnaRecord::reactivity @endlink.
+     *
+     * This member variable is used only if biological validated data is found (T.. fields in EBPSEQ are set).
+     *
+     * If @link RnaRecord::reactivity @endlink was derived as a consensus of different replicates,
+     * this indicates the standard error between samples used.
+     * If only one experiment was done, this may be some measure of variation between the data in that experiment,
+     * e.g. 0.2 of the standard deviation.
+     */
+    StringSet<String<float> > reactError;
+
+    /*!
+     * @var String<unsigned> RnaRecord::typeID
+     * @brief Indices of the assigned type (T..) attributes records from EBPSEQ files.
+     *
+     * This member variable is used only if biological validated data is found (T.. fields in EBPSEQ are set).
+     */
+    String<unsigned> typeID;
+
+    /*!
+     * @var Rna5String RnaRecord::sequence
+     * @brief String of bases in RNA strand.
+     *
+     * This member variable is only used in sequence-based records (from CT, DBN, DBV, BPSEQ, EBPSEQ files).
+     */
+    Rna5String sequence;
+
+    /*!
+     * @var StringSet<CharString> RnaRecord::seqID
+     * @brief Sequence identifier for aligned sequences.
+     *
+     * This member variable is only used in alignment-based records (from STH files).
+     */
+    StringSet<CharString> seqID;
+
+    /*!
+     * @var Align<Rna5String,ArrayGaps> RnaRecord::align
+     * @brief Alignment of several sequences (including gaps).
+     *
+     * This member variable is only used in alignment-based records (from STH files).
+     */
+    Align<Rna5String, ArrayGaps> align;
+
+    /*!
+     * @fn RnaRecord::hasUndefinedID()
+     * @brief Test for an undefined @link RnaRecord::recordID @endlink value.
+     * @return bool True if @link RnaRecord::recordID @endlink is not set.
+     * @signature bool RnaRecord::hasUndefinedID()
+     */
+    bool hasUndefinedID()
+    {
+        return recordID == UNDEF;
+    }
+
+    /*!
+     * @fn RnaRecord::clearID()
+     * @brief Clear value of @link RnaRecord::recordID @endlink and set to undefined.
+     * @signature void RnaRecord::clearID()
+     */
+    void clearID()
+    {
+        recordID = UNDEF;
+    }
+
+    /*!
+     * @fn RnaRecord::RnaRecord
+     * @brief The constructor.
+     * @signature RnaRecord::RnaRecord()
+     */
+    RnaRecord() : recordID(UNDEF), seqLen(0u), offset(1u), name(""), quality(""),
+                  comment(""), sequence("")
+    {}
+
```

Comment:
Use default member-initializer for offset and remove the def-constructor.
Default aggregate initialization should do the rest.